docs/ui/components/CollapsibleTalksGridWrapper/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from '@expo/styleguide';
+import { ArrowDownIcon } from '@expo/styleguide-icons/outline/ArrowDownIcon';
+import { useState } from 'react';
+
+import { Talk } from '~/public/static/talks';
+import { TalkGridWrapper, TalkGridCell } from '~/ui/components/Home/sections';
+
+type CollapsibleGridProps = {
+  items: Talk[];
+  initialCount?: number;
+};
+
+export function CollapsibleTalksGridWrapper({ items, initialCount = 8 }: CollapsibleGridProps) {
+  const [showAll, setShowAll] = useState(false);
+
+  return (
+    <>
+      <TalkGridWrapper>
+        {items.slice(0, showAll ? items.length : initialCount).map(item => (
+          <TalkGridCell key={item.videoId ?? item.event} {...item} />
+        ))}
+      </TalkGridWrapper>
+
+      {items.length > initialCount && !showAll && (
+        <div className="mt-6 flex justify-center">
+          <Button
+            size="lg"
+            theme="secondary"
+            onClick={() => {
+              setShowAll(true);
+            }}
+            rightSlot={<ArrowDownIcon className="icon-sm" />}>
+            Show More
+          </Button>
```

Comment:
```suggestion
          <Button
            theme="secondary"
            onClick={() => {
              setShowAll(true);
            }}>
            Show More
          </Button>
```

The down pointing arrow feels like the interaction would scroll the view, maybe let's stick to the label only. 

Also the button size seems a bit too big for an in-content us, usually those larger ones we use on marketing side. Would retain the default "md" sizing or even try to use `sm`.